samples/ScratchIslandApp/SampleApp/SampleAppLib.vcxproj

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{a4394404-37f7-41c1-802b-49788d3720e3}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>SampleApp</RootNamespace>
+    <ProjectName>SampleAppLib</ProjectName>
+    <TargetName>SampleAppLib</TargetName>
+    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>

```

Comment:
RS5? really?

Reply:
It's all copy-pasta from (in this case) TerminalAppLib.vcxproj, so I guess that's what it was targeting before